tests/image/test_push.py

```diff
@@ -15,6 +20,26 @@ def test_mock():
         yield m
 
 
+@pytest.fixture(autouse=True)
+def monkeypatch_bar_rate(monkeypatch):
```

Comment:
When I started writing new test I realized that converting output to regex manually is too much, I want something simpler.